### PR TITLE
Release v1.114.0 - release → staging

### PR DESCRIPTION
**Pull request by carrillo-erik**
### Added:

- Reintroduce NVMe Volume Upgrades ([#10229](https://github.com/linode/manager/pull/10229))

### Changed:

- Improve dev tools UI ([#10220](https://github.com/linode/manager/pull/10220))
- ACLB beta region from `Washington, DC` to `Miami, FL` ([#10232](https://github.com/linode/manager/pull/10232))

### Fixed:

- Incorrect units in Linode Network Graph Tooltip ([#10197](https://github.com/linode/manager/pull/10197))
- Disabled `Add` button once a node pool is added to kubernetes cluster in Create flow ([#10215](https://github.com/linode/manager/pull/10215))
- Invalid VPC scope with a Select All > Read Only selection in Create PAT drawer ([#10226](https://github.com/linode/manager/pull/10226))
- Disabled styles for Textfield input ([#10231](https://github.com/linode/manager/pull/10231))
- LinodeVolumeCreateForm crash ([#10235](https://github.com/linode/manager/pull/10235))

### Tech Stories:

- Update to React 18 ([#10169](https://github.com/linode/manager/pull/10169))
- Improve LinodeActionMenu restricted user experience ([#10199](https://github.com/linode/manager/pull/10199))
- Convert isRestrictedGlobalGrantType to Hook ([#10203](https://github.com/linode/manager/pull/10203))
- Update Storybook to latest to resolve CVE-2023-42282 ([#10212](https://github.com/linode/manager/pull/10212))
- Generate docs site sidebar based on folder structure ([#10214](https://github.com/linode/manager/pull/10214))
- Clean up `new QueryClient()` pattern in unit tests ([#10217](https://github.com/linode/manager/pull/10217))
- Remove build time API caching ([#10219](https://github.com/linode/manager/pull/10219))
- Clean up `Chip` component ([#10223](https://github.com/linode/manager/pull/10223))

### Tests:

- Add Cypress tests for account switching from Parent to Child ([#10110](https://github.com/linode/manager/pull/10110))
- Improve User Profile integration test coverage and separate from Display Settings coverage ([#10202](https://github.com/linode/manager/pull/10202))
- Add test for OBJ Multicluster bucket create flow ([#10211](https://github.com/linode/manager/pull/10211))
- Suppress Rollup warnings during Cypress tests ([#10239](https://github.com/linode/manager/pull/10239))

### Upcoming Features:

- Add list view for Linode Clone and Create from Backup ([#10182](https://github.com/linode/manager/pull/10182))
- Add ‘Delete Placement Group’ Modal (#10162)
- Update Placement Groups types, methods and factories (#10200)
- Add placement group details to Create Linode payload ([#10195](https://github.com/linode/manager/pull/10195))
- Update OBJ Multi-Cluster copy ([#10188](https://github.com/linode/manager/pull/10188))
- Handle errors gracefully when OBJ Multi-Cluster feature flag is enabled without MSW (#10189)
- Ensure correct ARIA labels for permissions are displayed in Access Key "Permissions" drawer when OBJ Multicluster is enabled ([#10213](https://github.com/linode/manager/pull/10213))
- Update Region Select for edge sites (#10194)
- Tag custom analytics events for account switching ([#10190](https://github.com/linode/manager/pull/10190))
- Improve Billing & Account restricted user experience ([#10201](https://github.com/linode/manager/pull/10201))
- Disable ability to edit or delete a proxy user via User Profile page ([#10202](https://github.com/linode/manager/pull/10202))
- Fix Users & Grants filtering error based on `user_type` ([#10230](https://github.com/linode/manager/pull/10230))
- Fix Account Switching ([#10234](https://github.com/linode/manager/pull/10234))
- Fix to ensure ChildAccountList receives proper account token (#10234)
- Rename database scale up to database resize ([#10193](https://github.com/linode/manager/pull/10193))

### Changed:

- Rename `database_scale` type to `database_resize` ([#10193](https://github.com/linode/manager/pull/10193))

### Upcoming Features:

- Accept placement group in Linode create payload ([#10195](https://github.com/linode/manager/pull/10195))

### Upcoming Features:

- Add Placement Group data in Create Linode payload ([#10195](https://github.com/linode/manager/pull/10195))
- Placement Group types update ([#10200](https://github.com/linode/manager/pull/10200))

### CODE REVIEW

**Review by jdamore-linode — APPROVED** [review]
It isn't showing up in the checks system for some reason (probably because of the issues we had Friday), but the Cypress tests _did_ pass for this PR

**Review by abailly-akamai — APPROVED** [review]

File: packages/validation/CHANGELOG.md [hunk 1]
```diff
@@ -1,36 +1,35 @@
-## [2024-02-13] - v0.40.0
```
Comment (bnussman-akamai) [inline packages/validation/CHANGELOG.md:1]: Not sure if I'm reading the diff wrong, but I think we may have lost this line

```
## [2024-02-13] - v0.40.0
```

**Review by mjac0bs — APPROVED** [review]
Confirmed version bumps, changelog versions and dates, and things were looking good in release testing.

File: packages/validation/CHANGELOG.md (hunk 1 above)
Comment (jdamore-linode) [reply packages/validation/CHANGELOG.md:1]: Ah yeah, confirmed that it's not there in my local copy

Comment (mjac0bs) [reply packages/validation/CHANGELOG.md:1]: Oh yeah, looks like it got updated to be today's release date instead. Good catch.

**Review by dwiley-akamai — APPROVED** [review]

File: packages/validation/CHANGELOG.md (hunk 1 above)
Comment (carrillo-erik) [reply packages/validation/CHANGELOG.md:1]: Whoops, seems like I deleted that line when I pasted the text for this release.

**Review by dwiley-akamai — APPROVED** [review]

**Review by abailly-akamai — APPROVED** [review]

**Review by bnussman-akamai — APPROVED** [review]
🚢 🚀 🎉

**Review by jdamore-linode — APPROVED** [review]

**Pull request merged**